tkm_circleCI_fieldTests/tkm_circleCI_fieldTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  tkm_circleCI_fieldTests.swift
+//  tkm_circleCI_field
+//
+//  Created by 長谷川 拓海(Takumi Hasegawa) on 2024/09/12.
+//  Copyright (c) 2024 *ReNKCHANNEL*. All rights reserved.
+//
+
+import XCTest
+
+final class tkm_circleCI_fieldTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/TakumiHasegawa21/tkm_circleCI_field/blob/cf59bd640260b9f05f2b0309a0ec1fa6f4c32ad7/tkm_circleCI_fieldTests/tkm_circleCI_fieldTests.swift#L11"></span>Type name 'tkm_circleCI_fieldTests' should only contain alphanumeric and other allowed characters<br />
<code>type_name</code> <code>tkm_circleCI_fieldTests.swift:11</code></td>
    </tr>
  </tbody>
</table>